providers/implementations/include/prov/implementations.h

```diff
@@ -277,6 +277,8 @@ extern const OSSL_DISPATCH ed25519_keymgmt_functions[];
 extern const OSSL_DISPATCH ed448_keymgmt_functions[];
 extern const OSSL_DISPATCH ec_keymgmt_functions[];
 extern const OSSL_DISPATCH kdf_keymgmt_functions[];
+extern const OSSL_DISPATCH mac_keymgmt_functions[];
```

Comment:
I think these names should indicate they are legacy.. by adding legacy as part of the name.